alarmpy/alarmpy.py

```diff
@@ -34,6 +36,11 @@ def __init__(
         self.repeat_alarms = repeat_alarms
         self.quiet = quiet
 
+        if desktop_notifications and not os.path.exists("/usr/bin/osascript"):
+            click.secho(f"Desktop notifications are currently only available for MacOS", fg="yellow")
```

Comment:
You can re-use `self.output_error()` here, and no need for the f-string

Reply:
done. f-string was there by mistake